Makefile

```diff
@@ -269,28 +269,6 @@ routes:
 		--output ./Sources/PointFreeRouter/__Generated__/DerivedPartialIsos.swift \
 		&& echo "  ✅ Generated!"
 
-SOURCERY_TESTS_IMPORTS = \
-	@testable import FunctionalCssTests; \
-	@testable import DatabaseTests; \
-	@testable import GitHubTests; \
-	@testable import ModelsTests; \
-	@testable import PointFreeRouterTests; \
-	@testable import PointFreeTests; \
-	@testable import StripeTests; \
-	@testable import StyleguideTests; \
-	@testable import SyndicationTests;
-
-sourcery-tests: check-sourcery
```

Comment:
I think we can drop Sourcery completely soon...

Reply:
🙏